parl/remote/scripts.py

```diff
@@ -133,11 +137,21 @@ def start_master(port, cpu_num, monitor_port, debug):
     # Redirect the output to DEVNULL to solve the warning log.
     _ = subprocess.Popen(
         master_command, stdout=FNULL, stderr=subprocess.STDOUT)
+
+    # Sleep 1s for master ready
+    time.sleep(1)
```

Comment:
move this line to the following `if` field.

Reply:
done